db/schema.rb

```diff
@@ -82,7 +82,7 @@
 
   add_index "journals", ["user_id"], name: "index_journals_on_user_id", using: :btree
 
-  create_table "orders", force: true do |t|
+  create_table "orders", force: :cascade do |t|
```

Comment:
[rubocop] Prefer single-quoted strings when you don't need string interpolation or special symbols.